package.json

```diff
@@ -54,7 +54,7 @@
     },
```

Comment:
cssnano will need to be bumped to resolve https://github.com/geocortex/vertigis-web-samples/security/dependabot/yarn.lock/normalize-url/open

Reply:
Thanks. I will bump the version (along with some others), however it appears that there is also a different vulnerability even in the latest version:

![image](https://user-images.githubusercontent.com/4603335/121934637-6a2b9c80-ccfc-11eb-81ef-54ae2905b001.png)

I'm not overly concerned about this, since this will only be used at build time and doesn't affect the security of the library code that is ultimately produced by the SDK.

Reply:
Turns out this broke the build, since the latest cssnano doesn't work with our older webpack 4 config and plugins. Updating to webpack 5 is out of scope for this PR, so I just reverted the change for now.